lib/uniform/modifiers.ex

```diff
@@ -8,7 +8,7 @@ defmodule Uniform.Modifiers do
   # Eject Fences
   #
 
-  @doc """
+  @doc ~S"""
```

Comment:
It turns out the docs rendered the examples incorrectly because the escape sequences in regexes were applied before rendering the docs.

(`.rs` instead of `\.rs`; `/\*` instead of `/\\*`; etc.)